test-cypress/cypress/commands/api/recording.ts

```diff
@@ -118,12 +175,19 @@ function addTracksToRecording(
     tracks: []
   };
 
+  if (tags && !trackDetails) {
+    trackDetails = tags.map((tag) => {
```

Comment:
An ES6 shorthand for this is `tags.map((tag) => ({tag}))`

Reply:
Thanks for this tip!